src/components/WheelCanvas/index.tsx

```diff
@@ -134,6 +134,9 @@ const drawWheel = (
       ctx.stroke();
 
       // TEXT FILL
+      ctx.font = `bold ${((style && style.fontSize) || fontSize) * 2}px ${
```

Comment:
Change to `${((style?.fontSize) || fontSize) * 2}`

Reply:
And check if the `bold` string changes something, check if it's necessary or not